collections/app/model/Node.scala

```diff
@@ -22,8 +22,7 @@ object Node {
           // use the T at this level or an empty node to hold children
           Node(currentSlug, loop(children, thisFullPath), thisFullPath, thisLevel.headOption)
         }
-        .sortBy(_.fullPath.head)
-        .sortBy(_.children.isEmpty)
+        .sortBy(node => (node.children.isEmpty, node.basename))
```

Comment:
[I sort of kind of do](https://joelneely.wordpress.com/2008/03/29/sorting-by-schwartzian-transform-in-scala/).

In short, `sortBy` has a nice little trick built in when you sort by a tuple.
[Here's a simpler, practical example](http://stackoverflow.com/questions/10027543/how-to-sort-a-list-in-scala-by-two-fields).

If you're wandering about the actual reason for the parameters, it's sorted by if the collections has children, and then alphabetically, similar to file system UIs. So the collapsable ones are always at the top.

Slightly ironic screengrab:
![nochildren](https://cloud.githubusercontent.com/assets/31692/12329899/74192bd4-bada-11e5-9bef-c058dee0f4f9.png)